spotbugs/src/main/java/edu/umd/cs/findbugs/cwe/WeaknessCatalog.java

```diff
@@ -16,13 +19,17 @@
 import com.google.gson.JsonSyntaxException;
 import com.google.gson.stream.JsonReader;
 
+import edu.umd.cs.findbugs.AbstractBugReporter;
+
 /**
  * The weakness catalog contains a number of weaknesses
  *
  * @author Jeremias Eppler
  * @see Weakness
  */
 public class WeaknessCatalog {
+    private static final Logger LOG = LoggerFactory.getLogger(AbstractBugReporter.class);
```

Comment:
Wrong class. Perhaps use `LoggerFactory.getLogger(MethodHandles.lookup().lookupClass())` to avoid this issue?

Reply:
seems expensive doing that nevermind fact slf4j will let you provide nothing now (at least in version 2).  Personally just fix it to correct class name WeaknessCatalog.class as many scanning tools expect that.

I don't know about other code in spotbugs, but upper case LOG is long incorrect.  It should simply be 'logger' in lower case as the accepted practice.

Reply:
It's a static final field. The [Google Java style guide](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names) still says to use UPPER_SNAKE_CASE for those, and so does [Carnegie Mellon](https://www.cs.cmu.edu/~rdriley/121/resources/styleguide/#s5.2.4-constant-names), and also [Oracle](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html). What Java style guide says otherwise?

Reply:
To my response, see https://itecnote.com/tecnote/java-should-a-static-final-logger-be-declared-in-upper-case/

Main point as many state similar and in fact slf4j states to use it that way is that its a reference not a constant.

Reply:
The linked page doesn't seem to give any actual basis for treating static final references differently from constants.

Reply:
@ThrawnCA Please mark as ok here as it was fixed.  Noted few times here on why 'logger', please look closer at the documentation you referenced even.  Feel like this review went south here but hopefully it was worth learning for all :)

I'm ok with the PR at this point, wanting your final opinion on PR itself now so we can finish this off.